src/data/complex/module.lean

```diff
@@ -237,6 +237,18 @@ begin
   fin_cases i; fin_cases j; simp
 end
 
+/-- The identity and the complex conjugation are the only two `ℝ`-algebra homomorphisms of `ℂ`. -/
+lemma real_alg_hom (f : ℂ →ₐ[ℝ] ℂ) : f = alg_hom.id ℝ ℂ ∨ f = conj_ae :=
```

Comment:
The name should be something like `real_alg_hom_eq_or_eq` or `real_alg_hom_eq_id_or_conj`.